samples/greengrass/basic_discovery/main.cpp

```diff
@@ -19,7 +19,7 @@
 using namespace Aws::Crt;
 using namespace Aws::Discovery;
 
-int main(int argc, char *argv[])
+int main(const int argc, const char *argv[])
```

Comment:
Sorry, I wasn't clear in the feedback.  The main()s were fine as is.  It was the function you made that took two char ** that should have been const (if it still exists).

Reply:
Oh, my bad! I adjusted the code so it no longer sets `main` to const while still keeping the function that takes two char** (`SendArguments`) const.